sellotape/sellotape_dj/main_app/models.py

```diff
@@ -1,8 +1,32 @@
 from django.db import models
 from django.contrib.auth.models import User
 
+# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+class Profile(models.Model):
+    # Inherits Attributes from User default instance
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    # User avatar, optional: limit the image instance height and width
+    avatar = models.ImageField()
+    country = models.CharField(max_length=100, default='Israel')
+    
+    class City(models.IntegerChoices):
+        tel_aviv = '1', 'Tel Aviv'
+        hamerkaz = '2', 'HaMerkaz'
+        hadarom = '3', 'HaDarom'
+        hatzafon = '4', 'HaTzafon'
+    
+    city = models.IntegerField(
+        choices=City.choices,
+        default=City.tel_aviv,
+    )   
+
+class UserFollowers(models.Model):
```

Comment:
the names of the field have to be more clarified, maybe use _'followers'_ and _'following'_ like Instagram's fields.

Reply:
I think it can make some misunderstanding as it supposed to be `follow_id` which is the user follows, and `following_id` as the followed user and it seems to be confusing, rather than `follower_id` and `user_id` respectively.